test/integration/spec/connect.js

```diff
@@ -990,6 +1013,73 @@ describe('connect', function() {
           assert.equal(loPriTrackPub.track.isSwitchedOff, true, 'low pri tracks were not switched off');
           assert.equal(hiPriTrackPub.track.isSwitchedOff, false, 'hi pri track were switched off');
         });
+
+        it('should switch off RemoteVideoTracks that are not published by the Dominant Speaker', async () => {
+          const [aliceTracks, bobTracks] = await Promise.all([1, 2].map(async () => [
+            createSyntheticAudioStreamTrack() || await createLocalAudioTrack({ fake: true }),
+            await createLocalVideoTrack(smallVideoConstraints)
+          ]));
+
+          // Initially disable Alice's audio
+          aliceTracks[0].enabled = false;
+
+          const [aliceLocal, bobLocal] = thoseRooms.map(room => room.localParticipant);
+          const [aliceRemote, bobRemote] = [thisRoom.participants.get(aliceLocal.sid), thisRoom.participants.get(bobLocal.sid)];
+
+          // Alice and Bob publish their LocalTracks
+          await Promise.all([
+            aliceLocal.publishTracks(aliceTracks),
+            bobLocal.publishTracks(bobTracks),
+            tracksSubscribed(aliceRemote, 2),
+            tracksSubscribed(bobRemote, 2)
+          ]);
+
+          // Bob should be the Dominant Speaker and only his RemoteVideoTracks
+          // should be switched on.
+          await Promise.all([
+            dominantSpeakerChanged(thisRoom, bobRemote),
+            ...[...bobRemote.videoTracks.values()].map(({ track }) => trackSwitchedOn(track)),
```

Comment:
NIT: instead of inlining the track finding code, I would put it out in a separate variable
```
const bobVideoTrack = Array.from(bobRemote. videoTracks.values())[0].track;
```
and use it here.